src/mqtt.cpp

```diff
@@ -128,15 +128,20 @@ make_value_path(const string& prefix, const OpenZWave::ValueID& v)
     if (!n->location.empty()) {
         name_path += n->location + "/";
     }
-    name_path += n->name + "/" + command_class_str(cmd_class);
-    id_path += to_string(n->id) + "/" + to_string(cmd_class);
+
+    // Always include instance
+    name_path += n->name + "/" + to_string(v.GetInstance()) + "/" + command_class_str(cmd_class);
+    id_path += to_string(n->id) + "/" + to_string(v.GetInstance()) + "/" + to_string(cmd_class);
 
     // Several command types support multi instances (e.g. 2-relay binary switch), so, add instance as well
     // Today we've found only 2 types that support it: SWITCH_MULTILEVEL and SWITCH_BINARY
+    /*
```

Comment:
Please remove commented out code since we have `git` for tracking history.

Reply:
I will.
Before I will make any changes, I would like to agree with You topic structure, because I fill that we have a little different idea / expectations: 
Topic structure:
[<Prefix>/][<Location>/]<Name>/<Instance number>/<Command Class>/<Value Label>
or
[<Prefix>/]<Node ID>/<Instance number>/<Command Class Code>/<Value ID>

If I properly understand, Your idea have been:
[<Prefix>/][<Location>/]<Name>/<Command Class>/<Instance number>/<Value Label>
(different hierarchy level)
(I preffer clear herarchy, from bottom to top)

Reply:
So yeah, original idea was:
```
[Prefix/][Location/]<Name>/<CommandClass>/<Instance Number>/<ValueLabel>
```
It was taken from `openzwave` config file:
```xml
<Node ...>
    <CommandClasses>
        <CommandClass id="38" name="COMMAND_CLASS_SWITCH_MULTILEVEL" ...>
            <Instance index="1" />
            <Instance index="2" endpoint="1" />
            <Value type="byte" ... />
        </CommandClass>
```
I'd prefer to leave current topic structure because if you change it completely it will affect all existing deployments, comparing to only single-instance devices will be affected in each deployment.